Assets/Scripts/Board/Board.cs

```diff
@@ -216,7 +224,96 @@ private void UpdateBoardUI()
 
     private void UpdateBoardGraph()
     {
+        Dictionary<TileDirection, List<(Graph.VertexDirection, Graph.VertexDirection)>> vertexMappings = new() {
+            { TileDirection.Northwest, new() { ( Graph.VertexDirection.Northwest, Graph.VertexDirection.South ),
+                                            ( Graph.VertexDirection.North, Graph.VertexDirection.Southeast ) } },
+            { TileDirection.Northeast, new() { ( Graph.VertexDirection.North, Graph.VertexDirection.Southwest ),
+                                            ( Graph.VertexDirection.Northeast, Graph.VertexDirection.South ), } },
+            { TileDirection.West, new() { ( Graph.VertexDirection.Northwest, Graph.VertexDirection.Northeast ),
+                                        ( Graph.VertexDirection.Southwest, Graph.VertexDirection.Southeast ), } },
+            { TileDirection.East, new() { ( Graph.VertexDirection.Northeast, Graph.VertexDirection.Northwest ),
+                                        ( Graph.VertexDirection.Southeast, Graph.VertexDirection.Southwest ), } },
+            { TileDirection.Southwest, new() { ( Graph.VertexDirection.Southwest, Graph.VertexDirection.North ),
+                                            ( Graph.VertexDirection.South, Graph.VertexDirection.Northeast ), } },
+            { TileDirection.Southeast, new() { ( Graph.VertexDirection.Southeast, Graph.VertexDirection.North ),
+                                            ( Graph.VertexDirection.South, Graph.VertexDirection.Northwest ), } },
+        };
```

Comment:
Does this need to be inside the function or can it be an external dictionary like the others

Reply:
Yeah, we can have it be a static dict in the Graph